src/app/login/login.component.html

```diff
@@ -11,21 +11,21 @@
     </div><!--/.col-*-->
 
     <div class="col-sm-7 col-md-6 col-lg-5 login">
-      <form class="form-horizontal" role="form">
+      <form id="githubLogin" class="form-horizontal" role="form">
```

Comment:
I suggest to have the keyword "form" in the id. For an example "githubLoginForm".